components/sponsored_benefits/app/controllers/sponsored_benefits/organizations/plan_design_proposals_controller.rb

```diff
@@ -32,6 +44,7 @@ def create
 
       respond_to do |format|
         if proposal_form.save
+          proposal_form.publish! # is this the correct place to publish?
```

Comment:
no we want them to be able to create / save / edit a proposal potentially several times before they choose to publish